packages/admin-ui/src/combobox/combobox-multiple-field.tsx

```diff
@@ -51,6 +52,15 @@ export const ComboboxMultipleField = createComponent<
     ...htmlProps
   } = props
 
+  const {
+    selectedItems,
+    value: inputValue,
+    removeSelectedItem,
+    clearSelected,
+    getOptionValue,
+    renderTag,
+  } = state
```

Comment:
I would keep this in the `state` variable. It is easier to read and you don't need to rename things